File: URI/1 - INICIANTE/Python/1182 - ColunaNaMatriz.py
l = int(input())
operacao = input()
matriz = []
soma = 0.0
for x in range(0, 12):
    linha = []
    for y in range(0, 12):
        linha.append(float(input()))
    matriz.append(linha)
for x in range(0, 12):
    soma += matriz[x][l]
if operacao == 'S':
    print('{:.1f}'.format(soma))
else:
    print('{:.1f}'.format(soma/12))

File: URI/1 - INICIANTE/C/1182 - ColunaNaMatriz.c
#include <stdio.h>

int main(){
	double matriz[12][12], soma = 0;
	char op;
	int x, linha = 0, y;
	scanf(" %d", &linha);
	scanf(" %c", &op);
	for(x = 0; x < 12; x++){
		for(y=0;y < 12; y++){
			scanf(" %lf", &matriz[x][y]);
		}
	}
	for(x = 0; x < 12; x++){
		soma += matriz[x][linha];
	}
	if(op == 'S'){
		printf("%.1lf\n", soma);
	}else{
		printf("%.1lf\n", soma/12);
	}
	return 0;
}
